### ISSUE TITLE
AI target priorities 

### ISSUE THREAD

**Issue by rune-b** [Sandrem/FlyCasual#3011]
1.6.1

AI M3-A shot at interceptor with evade at range 3 instead of interceptor with focus at range 1

**Comment by Sandrem** [comment]
Maybe this bug is related to installed Heavy Laser Cannon with range 2-3

**Comment by rune-b** [comment]
Yeah, that could be. Maybe it doesn't check properly for bullseye
